### PR TITLE
Coreyching/update super todo

### PR DESCRIPTION
**Pull request by cching-stripe**
## Summary

Updating the Super Todo app to match the designs set in our Figma and Docs. Adding in a 2 panel design and a toggle button in Todos panel. 

## Motivation

Improving Super Todo usability. Simplifying the example app for future developers.

### CODE REVIEW

File: examples/super-todo/src/views/CustomerListView.tsx [hunk 1]
```diff
@@ -115,58 +121,11 @@ const CustomerListView = () => {
...
-                    }
-                  />
-                </Box>
-                <Box css={{ marginTop: "small" }}>
-                  <Divider/>
-                </Box>
-              </Box>
-            </TabPanel>
+            {/* Render Todos */}
+            {showTodos(showCompletedTodos, setTodoType, showCompletedTodos)}
```
Comment (charlesw-stripe) [inline examples/super-todo/src/views/CustomerListView.tsx:125]: Looks like showTodos only takes in the first two arguments

**Review by ericfrank-stripe — APPROVED** [review]

File: examples/super-todo/src/views/CustomerListView.tsx [hunk 2]
```diff
@@ -36,7 +32,18 @@ const lists = {
...
 };
-const yourLists = ["pear", "orange", "melon"];
+
+const todos = {
+  "janefisher@gmail.com": { task: "Respond to", assignee: "Tayler", checked: false},
+  "ryan.lee@gmail.com": { task: "Respond to", assignee: "Tayler", checked: false},
+  "john.ryan@cloudly.com": {task: "Send payout to", assignee: "Corey", checked: false}
+};
+
+const completed_todos = {
```
Comment (charlesw-stripe) [inline examples/super-todo/src/views/CustomerListView.tsx:42]: Nit: camel-case, completedTodos

**Review by charlesw-stripe — COMMENTED** [review]
Some small nits but LGTM

File: examples/super-todo/src/views/CustomerListView.tsx (hunk 1 above)
Comment (cching-stripe) [reply examples/super-todo/src/views/CustomerListView.tsx:125]: Good catch. Done.

File: examples/super-todo/src/views/CustomerListView.tsx (hunk 2 above)
Comment (cching-stripe) [reply examples/super-todo/src/views/CustomerListView.tsx:42]: done.

File: examples/super-todo/src/components/TodosComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  Checkbox,
+  Divider,
+  Switch
+} from "@stripe/ui-extension-sdk/ui";
+
+type TodosComponentProps = {
+  todos: object,
+  setTodoType: Function,
+  showCompletedTodos: Boolean
+};
```
[the remark this answers is not in the document]
Comment (cching-stripe) [reply examples/super-todo/src/components/TodosComponent.tsx:13]: Good suggestion - done.

**Pull request merged**